### PR TITLE
Fix L0_backend_python/env test

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

**Review by nv-kmcgill53 — DISMISSED** [review]

**Review by krishung5 — COMMENTED** [review]
Nit: update the copyright for `test.sh`. LGTM otherwise!

File: qa/L0_backend_python/env/test.sh [hunk 1]
```diff
@@ -83,20 +86,20 @@ cp ../../python_models/python_version/model.py ./models/python_3_6/1/
...
-conda install tensorflow=2.3.0 -y
-PY38_VERSION_STRING="Python version is 3.8, NumPy version is 1.19.1, and Tensorflow version is 2.3.0"
-conda-pack -o python3.8.tar.gz
-mkdir -p models/python_3_8/1/
-cp ../../python_models/python_version/config.pbtxt ./models/python_3_8
-cp python3.8.tar.gz models/python_3_8/python_3_8_environment.tar.gz
-(cd models/python_3_8 && \
-          sed -i "s/^name:.*/name: \"python_3_8\"/" config.pbtxt && \
+path_to_conda_pack='$$TRITON_MODEL_DIRECTORY/python_3_10_environment.tar.gz'
+create_conda_env "3.10" "python-3-10"
```
[the remark this answers is not in the document]
Comment (Tabrizian) [reply qa/L0_backend_python/env/test.sh:92]: Added a comment.

**Review by krishung5 — APPROVED** [review]

**Review by tanmayv25 — DISMISSED** [review]

File: qa/L0_backend_python/common.sh [hunk 2]
```diff
@@ -31,7 +31,7 @@ get_shm_pages() {
 
 install_conda() {
   rm -rf ./miniconda
-  file_name="Miniconda3-py310_23.1.0-1-Linux-x86_64.sh"
+  file_name="Miniconda3-py310_23.3.1-0-Linux-x86_64.sh"
```
Comment (mc-nv) [inline qa/L0_backend_python/common.sh:34]: @Tabrizian does that value somehow interfere, do we need to change the `build.py` version as well?
https://github.com/triton-inference-server/server/blob/r23.05/build.py#L77

Comment (Tabrizian) [reply qa/L0_backend_python/common.sh:34]: It doesn't interfere. It is just a local installation of miniconda.

Comment (Tabrizian) [reply qa/L0_backend_python/common.sh:34]: We can use a different version from `build.py` miniconda.

**Review by krishung5 — APPROVED** [review]

**Pull request merged**